storage/conf/spanner_conf.go

```diff
@@ -0,0 +1,12 @@
+package conf
+
+type SpannerConfig struct {
+	Project  string `mapstructure:"project" validate:"required"`
+	Instance string `mapstructure:"instance" validate:"required"`
+	Database string `mapstructure:"db" validate:"required"`
+
+	S3Bucket  string `mapstructure:"s3_bucket"`
+	S3FileKey string `mapstructure:"s3_file_key"`
+	// for local test
+	CredentialsFile string `mapstructure:"credentials_file"`
```

Comment:
```suggestion
	CredentialsFile string `mapstructure:"credentials_file"`
```
We only care where are the credentials file and the s3 path makes no sense here.

Reply:
Ok, will remove them.